src/core/page/PageDelete.tsx

```diff
@@ -10,9 +10,13 @@ const PageDelete: React.FC<{ onClose: () => void, articleId: StencilClient.Artic
   const { enqueueSnackbar } = useSnackbar();
   const { service, actions, site } = Composer.useComposer();
   const [pageId, setPageId] = React.useState('');
-
+  const tabs = Burger.useTabs();
 
   const handleDelete = () => {
+    var pageTab = tabs.session.tabs.find(tab => tab.id === props.articleId)
+    if (pageTab) {
```

Comment:
Line 17-19 should be moved into the Promise response. This way, the tab is closed only after successful deletion.